spec/models/endpoint_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Endpoint, type: :model do
+
+  let!(:endpoint) { Endpoint.create(endpoint_name: 'aws', endpoint_type: 'cloud') }
+
+  it 'is not valid without valid attributes' do
```

Comment:
Can we add another test where just 1 of the 2 requirements are fulfilled? Or am I too anal?

Reply:
Did this get addressed by added specs below?